src/OrbitGl/TimerChain.h

```diff
@@ -108,6 +114,7 @@ class TimerChain {
   ~TimerChain();
 
   void push_back(const TextBox& item) { current_->Add(item); }
+  [[nodiscard]] TextBox* Last() { return current_->Last(); }
```

Comment:
Nit: Other methods that return a TextBox* are called `GetSomething`. Call this `GetLast`?

Reply:
Yes, thanks. I also moved the function alongside the other getters.